ngraph/frontend/onnx_import/src/core/node.cpp

```diff
@@ -90,36 +94,37 @@ namespace ngraph
             return it != std::end(m_attributes);
         }
 
-        template <typename T>
-        T Node::Impl::get_attribute_value(const std::string& name, T default_value) const
+        Subgraph Node::Impl::get_subgraph_from_attribute(
+            const std::string& name,
+            const std::map<std::size_t, element::Type_t>& subgraph_inputs_types_map) const
         {
             auto it = std::find_if(
                 std::begin(m_attributes), std::end(m_attributes), [&](const Attribute& attribute) {
                     return attribute.get_name() == name;
                 });
             if (it == std::end(m_attributes))
             {
-                return std::forward<T>(default_value);
+                throw error::node::UnknownAttribute{this->name(), name};
```

Comment:
Not sure if this is already used in this file but we have macros for validation: NGRAPH_CHECK and NODE_VALIDATION_(something)

Reply:
There is no change in this line - this is a diff tool "bug". Do you want me to refactor it in this PR?

Reply:
Not necessarily, we can leave it as is.